Posts/2021/10/crescendo-output-handler.md

```diff
@@ -0,0 +1,145 @@
+---
+Categories: PowerShell
+post_title: A closer look at the parsing code of a Crescendo output handler
+Summary: In this post I take a close look at one of my output handler and talk about the different parsing methods I used.
+tags: Crescendo, output handler, parsing
+username: sewhee@microsoft.com
+---
+In my [previous post][2], I showed you how to parse the output from the `netstat` command. The
+output of `netstat` is not very complex. The goal of the post was to introduce some parsing
+strategies that you can use to create a full Crescendo module. In this post, I explain the details
+of a more complex parsing function that I created for my [VssAdmin module][7].
+
+## Examining the parser for `Get-VssShadowStorage`
+
+The following screenshot provides line numbers for the `ParseShadowStorage` function. This is the
+function called by `Get-VssShadowStorage` to handle the output (parse) of the `vssadmin.exe`
```

Comment:
```suggestion
parsing function called by `Get-VssShadowStorage` to handle the output of the `vssadmin.exe`
```

just a suggestion. Up to you if you want to include it.

Reply:
Yep, fixed already.